department-of-veterans-affairs/abd-vro
# Chengjie8/issue 2724

## What was the problem?
- kafka topic names have recently been updated by BIE team, so need to correct that in the `BieMessagePayload` java class
- added the migration script for the new column in RDS

Associated tickets or Slack threads:
- #2724


## How to test this PR
- verify in RDS


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
